partiql-plan/src/main/kotlin/org/partiql/plan/v1/rex/RexCast.kt

```diff
@@ -0,0 +1,13 @@
+package org.partiql.plan.v1.rex
+
+import org.partiql.types.PType
+
+/**
```

Comment:
What's the sequencing for the PRs? Saw that this was marked as 1/2 in the title. Will `Rel`s be in a different PR? Similarly for the visitors + builders?

Reply:
Correct, rels will be another **commit**.  Visitors/builders will be another **pull request**.